src/pattern-library/patterns/authors/authors.njk

```diff
@@ -1,5 +1,5 @@
-<div class="w-author">
-  <a href="{{ data.url }}">
+<div class="author">
+  <a class="avatar" href="{{ data.url }}">
     <img
```

Comment:
I know this isn't part of the review, why not use our shortcode here?

Reply:
Not that part of the project yet and the current shortcodes still have a lot of hangover from the older stuff (hence the `w-` prefix)